viz.yaml

```diff
@@ -0,0 +1,81 @@
+vizlab: "0.3.6"
+info:
+  id: gage-conditions
+  name: USGS Gage Conditions
+  date: 2018-06-11
+  publish-date: 2019-06-25
+  path: /gage-conditions
+  analytics-id: # WE WILL NEED TO FILL THIS IN AT SOME POINT
+  description: Map of all USGS gages and their conditions.
+  required-packages:
+    vizlab:
+      repo: github
+      version: 0.3.6
+      name: USGS-VIZLAB/vizlab
+      ref: v0.3.6
+    dataRetrieval:
+      repo: CRAN
+      version: 2.7.4
+    dplyr:
+      repo: CRAN
+      version: 0.7.4
+    tidyr:
+      repo: CRAN
+      version: 0.8.1
+  contributors:
+    -
+      name: Lindsay Carr
+      email: lcarr@usgs.gov
+      affiliation: U.S. Geological Survey
+    -
+      name: Jordan Read
+      email: jread@usgs.gov
+      affiliation: U.S. Geological Survey
+parameter:
+  - id: plot_monthday
+    monthday: "05-01"
+  - id: plot_year
+    year: 2018
+  - id: block_size
+    size: 10
+  - id: discharge_percentiles
+    percentiles: ["05","10","20","25","50","75","80","90","95"]
```

Comment:
Should these be numeric? (and coerced into char as needed if they need to be chars?)

Reply:
I needed them to be characters more than I needed them to be numeric, so I chose to make them character by default. Is that good logic?

Reply:
Sounds good :+1: